static/css/lib/custom-bulma.css

```diff
@@ -2239,7 +2239,8 @@ div.icon-text {
   color: #fff;
 }
 .table td.is-warning,
-.table th.is-warning {
+.table th.is-warning,
+.table tr.is-warning {
```

Comment:
We don't want to edit this file because it's compiled from the `scss/custom-bulma.sass` file (compilation steps are documented at the top of that file). General rule of thumb is if we're adding/changing a style that modifies Bulma's out of the box styles, we edit it there. Otherwise we add whichever styles we need to the `<style>` tag at the end of the template files.

Reply:
Fair point, this was actually changed when I was trying out different styles, it's not needed at all.